libs/converter-nh-accuvote/src/convert/convert_election_definition.ts

```diff
@@ -277,6 +293,8 @@ async function convertCardDefinition(
           ),
         },
       ],
+      },
```

Comment:
This looks like the auto-formatting didn't run here - I would expect the contents of this object literal to be indented